Gemfile

```diff
@@ -25,6 +25,7 @@ gem 'redis', '~> 4.0'
 # gem 'rack-cors', '~>5.2.7'
 gem 'active_model_serializers', '~> 0.10.0'
 gem 'devise_token_auth'
+gem 'httparty'
```

Comment:
This gem was installed in the previous PR. You should change your base branch :wink: